index.d.ts

```diff
@@ -67,15 +72,15 @@ export class FlatGrid<ItemType = any> extends React.Component<
 > {}
 
 export interface SectionGridProps<ItemType = any> {
-  renderItem: (info: { item: SectionListData<ItemType> }) => JSX.Element;
-  sections: ItemType,
-  itemDimension?: number,
-  fixed?: boolean,
-  spacing?: number,
-  style?: StyleProp<ViewStyle>,
-  staticDimension?: number,
-  renderSectionHeader?: (info: { section: SectionListData<ItemType> }) => JSX.Element;
-  onLayout?: func,
+  renderItem: (info: ListRenderItemInfo<ItemType>) => JSX.Element;
+  sections: ItemType;
+  itemDimension?: number;
+  fixed?: boolean;
+  spacing?: number;
+  style?: StyleProp<ViewStyle>;
+  staticDimension?: number;
+  renderSectionHeader?: (info: SectionListData<ItemType>) => JSX.Element;
```

Comment:
I think `renderSectionHeader?: (info: { section: SectionListData<ItemType> })` was correct here. That was taken from the type definitions of native SectionList